packages/website/src/components/Uploader/Uploader.js

```diff
@@ -22,11 +22,11 @@ const getRootDirectory = (file) => {
 };
 
 const RegistrationLink = () => {
-  const accountsUrl = useAccountsUrl();
+  const createAccountsUrl = useAccountsUrl();
```

Comment:
Why is this function being assigned to a const? And why are we overriding the existing `createAccountsUrl` function? I don't know how React works but this looks really weird.

Reply:
I see that `createAccountsUrl` is actually returned from the `useAccountsUrl` function. It still looks weird. I also don't understand why we are assigning this to a variable.

Reply:
`useAccountsUrl` is a custom react hook https://reactjs.org/docs/hooks-intro.html and that's why it has to be assigned inside of the component